darts/backtesting/backtesting.py

```diff
@@ -268,7 +268,7 @@ def forecasting_residuals(model: ForecastingModel,
     first_index = series.time_index()[model.min_train_series_length]
 
     # compute fitted values
-    p = backtest_forecasting(series, model, first_index, fcast_horizon_n, True, verbose=verbose)
+    p = backtest_forecasting(series, model, first_index, fcast_horizon_n, trim_to_series=True, verbose=verbose)
```

Comment:
should this be a part of this PR?

Reply:
It is a part of fixing the develop branch after searching the cause of bugs. But this is not caused by the PR #123 
So yes, it doesn't belong to this PR

Reply:
Wait so where exactly did this cause a bug? Because shouldn't line 274 make sure that the two series have the same indices anyways? I mean I think the change is good, because it should be (very) slightly more efficient. But I believe it would be good to have clearer scopes in individual PRs (don't get me wrong though, I'm guilty of this too).

Reply:
My bad, there is no bug, because this function is only for univariate time series. I revert it.
Nonetheless, this parameter shouldn't be here, even if it does nothing for now.